optuna/samplers/_cmaes.py

```diff
@@ -131,10 +154,20 @@ def __init__(
         self._cma_rng = np.random.RandomState(seed)
         self._search_space = optuna.samplers.IntersectionSearchSpace()
         self._consider_pruned_trials = consider_pruned_trials
+        self._restart_strategy = restart_strategy
+        self._inc_popsize = inc_popsize
 
```

Comment:
Then, we can use the defined function to raise an experimental warning when `restart_strategy != None`:
```suggestion

        if self._restart_strategy:
            self._raise_experimental_warning_for_restart_strategy()

```